tests/all/main.rs

```diff
@@ -209,8 +196,150 @@ impl Project {
         self
     }
 
+    fn generate_bindings(&mut self) -> Vec<PathBuf> {
```

Comment:
Nitpick: to be a little more clear, I think we should name this `generate_webidl_bindings`, since this whole project involves bindings of some sort